src/api/realmDataTypes.js

```diff
@@ -118,6 +118,8 @@ export type RealmDataForUpdate = $ReadOnly<{
     $PropertyType<InitialDataRealm, 'realm_video_chat_provider'>,
   waiting_period_threshold:
     $PropertyType<InitialDataRealm, 'realm_waiting_period_threshold'>,
+  web_public_access_enabled:
+    $PropertyType<InitialDataRealm, 'realm_web_public_access_enabled'>,
```

Comment:
I don't see this in the API docs either.

This type is meant to describe:
```js
/**
 * Property-value map for data in `update` or `update_dict` realm events.
 *
 * Use this as a source where we define the actual types for those events.
 *
 * We use our type InitialDataRealm for how the data appears in the
 * /register response. The properties are named differently between here and
 * in the /register response (InitialDataRealm has lots of properties that
 * start with "realm_"). But we expect the values to be typed the same.
 */
/* prettier-ignore */
export type RealmDataForUpdate = $ReadOnly<{
```

So looking at the API docs for one of those:
https://zulip.com/api/get-events#realm-update_dict
there is no `web_public_access_enabled`.

There is one related-looking property, `enable_spectator_access`.